frontend/src/components/common/form/FastDatePicker.tsx

```diff
@@ -74,7 +75,7 @@ const FormikDatePicker: FunctionComponent<FastDatePickerProps> = ({
   if (typeof value === 'string') {
     value = moment(value, 'YYYY-MM-DD').toDate();
   }
-  if (value && value !== initialValue) {
+  if (value && dequal(moment(initialValue, 'YYYY-MM-DD'), moment(value, 'YYYY-MM-DD'))) {
```

Comment:
this is what was triggering all of the extra validations.

Reply:
Good catch